sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+
+**_Does the project augment or benefit other CNCF projects?_**
+
+## Anticipated use cases
+
+**_What problems does the project solve?_**
+
+## Alignment with SIG Reference Model
+
+**_Does the project align with the SIG CNCF reference model and which capabilities does it require/provide at each level of the reference model._**
+
+
+## High level architecture
+
+**_Describe the overall architecture of the project. Feel free to add diagrams._**
+
+# Formal Requirements
+
+**_Document that the project fulfills the requirements as documented in the [CNCF graduation criteria for sandbox](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#sandbox-stage)_**
+
+**_Are there any anticipated issues with any of the criteria ?_**
+
+
+# CNCF IP Policy
+
+**_Becoming a sandbox project requires adoption of the CNCF IP Policy: [https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy](https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy)_**
+
+**_Note: there is a grace period after becoming a sandbox period to enable projects to adopt the policy, however, some prep is required to ensure there are no major blockers._**
+
+**_Has the IP policy been reviewed?_**
+
+**_List the repos for the project and their current license_**
+
+**_List any dependent repos (upstream/downstream) that are required to build the project  (including but not limited to  libraries, commercial tools, plugins)_**
+
+**_What actions are required to be compliant with the IP policy?_**
+
+
+# Other Considerations
+
+## **_Please note, these are not gating criteria but rather to:_**
+
+
+*   **_Collect a standard set of information for each project_**
+*   **_Provides a point in time capture of the state of the project which makes it easier to track progress at future reviews and / or promotion_**
+*   **_Help projects to prepare for SIG and TOC presentation_**
+*   **_Allow the SIG to review the project and perform due diligence for incubation_**
+*   **_Provide the TOC with the information required to accept sponsorship of a project and/or votes_**
+*   **_Identify and rectify any significant issues / blockers prior to presenting to the TOC and acceptance as a CNCF project_**
+
+
+## Cloud Native
+
+**_Does the project meet the definition of Cloud Native?  The CNCF charter states:_**
+
+
+    “Cloud native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
+
+
+    “These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.”
+
+
+## Project and Code Quality
```

Comment:
Parts of this are covered in CII best practices.  Would be great to ask projects to fill out specific questions so this can be collected in standard format (that will then be linked from landscape)